index.js

```diff
@@ -60,7 +66,7 @@ var GulpJsonServer = function(options){
 
 		var server = jsonServer.create();
 
-		server.use(bodyParser.json());
+		server.use(bodyParser.json(this.options.bodyParser.json));
```

Comment:
@nedkelly I think it's better not to introduce bypassing bodyParser options through gjs, but rather introduce new option to provide own bodyParser if needed, and use it if it's provided. That way we are get rid from unnecessary fixes when new options for bodyParser are in need, and you will be able to tune parser as you want.